### ISSUE TITLE
Buttons don't have focus indicators 

### ISSUE THREAD

**Issue by carolstran** [electron/fiddle#1084]
While trying to tab around, I noticed that the buttons are lacking focus indicators. Not only does that make it really difficult to tell where on the page you are, but [it's also an accessibility issue](https://www.deque.com/blog/accessible-focus-indicators/) and could be unintentionally excluding people who are trying to use Fiddle.

Here's a screenshot from the Settings page with `:focus` styling enabled for all focusable elements:
![](https://user-images.githubusercontent.com/26869552/170577760-59aa3e8b-8981-4230-ba74-a2b447746f3f.png)
 
Your `main.less` file seems to be removing outline on focus and I'd guess this is where the problem is:
```
  *:focus {
    outline: none;
  }
```
